13-TeV-examples/uproot_python/HZZAnalysis.ipynb

```diff
@@ -533,9 +536,1417 @@
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.2s\n",
+      "Before the cut: [[-1, 1, 1, -1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[-1, 1, 1, -1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.2s\n",
+      "Before the cut: [[-1, 1, 1, -1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[-1, 1, 1, -1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.3s\n",
+      "Before the cut: [[1, -1, -1, -1]]\n",
+      "Risultato della condizione: [True]\n",
+      "After the cut: []\n",
+      "\t\t nIn: 1,\t nOut: \t0\t in 0.3s\n",
+      "Before the cut: [[-1, 1, 1, -1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[-1, 1, 1, -1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.3s\n",
+      "Before the cut: [[-1, -1, 1, 1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[-1, -1, 1, 1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.4s\n",
+      "Before the cut: [[-1, 1, -1, -1]]\n",
+      "Risultato della condizione: [True]\n",
+      "After the cut: []\n",
+      "\t\t nIn: 1,\t nOut: \t0\t in 0.4s\n",
+      "Before the cut: [[1, -1, -1, 1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[1, -1, -1, 1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.5s\n",
+      "Before the cut: [[-1, -1, 1, -1]]\n",
+      "Risultato della condizione: [True]\n",
+      "After the cut: []\n",
+      "\t\t nIn: 1,\t nOut: \t0\t in 0.6s\n",
+      "Before the cut: [[1, 1, -1, -1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[1, 1, -1, -1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.6s\n",
+      "Before the cut: [[-1, 1, 1, 1]]\n",
+      "Risultato della condizione: [True]\n",
+      "After the cut: []\n",
+      "\t\t nIn: 1,\t nOut: \t0\t in 0.6s\n",
+      "Before the cut: [[1, -1, 1, 1]]\n",
+      "Risultato della condizione: [True]\n",
+      "After the cut: []\n",
+      "\t\t nIn: 1,\t nOut: \t0\t in 0.7s\n",
+      "Before the cut: [[1, -1, 1, -1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[1, -1, 1, -1]]\n",
+      "\t\t nIn: 1,\t nOut: \t0\t in 0.7s\n",
+      "Before the cut: [[-1, -1, 1, 1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[-1, -1, 1, 1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.8s\n",
+      "Before the cut: [[-1, 1, 1, -1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[-1, 1, 1, -1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.8s\n",
+      "Before the cut: [[1, 1, -1, -1]]\n",
+      "Risultato della condizione: [False]\n",
+      "After the cut: [[1, 1, -1, -1]]\n",
+      "\t\t nIn: 1,\t nOut: \t1\t in 0.9s\n",
+      "Before the cut: [[-1, 1, -1, -1]]\n",
+      "Risultato della condizione: [True]\n",
+      "After the cut: []\n",
+      "\t\t nIn: 1,\t nOut: \t0\t in 0.9s\n",
+      "Before the cut: [[1, -1, 1, 1]]\n",
+      "Risultato della condizione: [True]\n",
```

Comment:
This is a lot of Italian :)

I imagine that you didn't really intend to update this notebook? Maybe reverting it is the way to go?

Reply:
Yeah, sorry about that!